doc/how-to/getting_started_python.rst

```diff
@@ -1,255 +1,263 @@
+    ..  code-block:: console
 
-.. code-block:: python
+        $ pip install tarantool
 
-    >>> tester.insert((4, 'ABBA', 1972))
-    [4, 'ABBA', 1972]
+4.  Inside the ``hello`` directory, create the ``hello.py`` file for application code.
 
-.. _getting_started-python-query:
 
-********************************************************************************
-Querying data
-********************************************************************************
+.. _getting_started_python_import_:
 
-Let's start with selecting a tuple by the primary key
-(in our example, this is the index named ``primary``, based on the ``id`` field
-of each tuple). Use ``select``:
+Importing 'tarantool'
+~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: python
+In the ``hello.py`` file, import the ``tarantool`` package:
 
-    >>> tester.select(4)
-    [4, 'ABBA', 1972]
+..  literalinclude:: /code_snippets/snippets/connectors/python/hello.py
+    :language: python
+    :start-at: import
+    :end-at: import
+    :dedent:
 
-Next, select tuples by a secondary key.
-For this purpose, you need to specify the number *or* name of the index.
 
-First off, select tuples using the index number:
+.. _getting_started_python_creating_connection:
 
-.. code-block:: python
+Connecting to the database
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    >>> tester.select('Scorpions', index=1)
-    [2, 'Scorpions', 2015]
+Add the following code:
 
-(We say ``index=1`` because index numbers in Tarantool start with 0,
-and we're using our second index here.)
+..  literalinclude:: /code_snippets/snippets/connectors/python/hello.py
+    :language: python
+    :start-at: # Connect to the database
+    :end-before: # Insert data
+    :dedent:
 
-Now make a similar query by the index name and make sure that the result
-is the same:
+This code establishes a connection to a running Tarantool instance on behalf of ``sampleuser``.
+The ``conn`` object can be used to make CRUD requests and execute stored procedures.
 
-.. code-block:: python
 
-    >>> tester.select('Scorpions', index='secondary')
-    [2, 'Scorpions', 2015]
 
-Finally, select all the tuples in a space via a ``select`` with no
-arguments:
+.. _getting_started_python_manipulating_data:
 
-.. code-block:: python
+Manipulating data
+~~~~~~~~~~~~~~~~~
 
-    >>> tester.select()
+.. _getting_started_python_inserting_data:
 
-.. _getting_started-python-update:
+Inserting data
+**************
 
-********************************************************************************
-Updating data
-********************************************************************************
+Add the following code to insert four tuples into the ``bands`` space:
 
-Update a :term:`field` value using ``update``:
+..  literalinclude:: /code_snippets/snippets/connectors/python/hello.py
+    :language: python
+    :start-at: # Insert data
+    :end-before: # Select by primary key
+    :dedent:
 
-.. code-block:: python
+``connection.insert()`` is used to insert a tuple to the space.
```

Comment:
```suggestion
``conn.insert()`` is used to insert a tuple to the space.
```

Reply:
or `Connection.insert()` if you refer to the library class instead of exact object in this example.
Same for occurrences of `connection.<method_name>` below in text.